js/flatbuffers.js

```diff
@@ -677,11 +677,11 @@ outer_loop:
  * @param {flatbuffers.Offset} root_table
  * @param {string=} opt_file_identifier
  */
-flatbuffers.Builder.prototype.finish = function(root_table, opt_file_identifier) {
+flatbuffers.Builder.prototype.finish = function(root_table, opt_file_identifier, size_prefix) {
```

Comment:
Is this function never called externally? Since we're requiring an extra arg now could be a breaking change?

Reply:
Should not be breaking, but I realize now it should probably be called opt_size_prefix and default to false to mimic the overloaded behaviour from the C#/Java impl.

Reply:
Fixed with https://github.com/google/flatbuffers/pull/5326/commits/ba77cc3eac427859d67515eba94674815b90e5dd.